src/routes/notifications/notifications.controller.spec.ts

```diff
@@ -237,4 +241,71 @@ describe('Notifications Controller (Unit)', () => {
         });
     });
   });
+
+  describe('DELETE /chains/:chainId/notifications/devices/:uuid/safes/:safeAddress', () => {
+    it('Success', async () => {
+      const uuid = faker.datatype.uuid();
+      const safeAddress = faker.finance.ethereumAddress();
+      const chain = chainBuilder().build();
+      const expectedProviderURL = `${chain.transactionService}/api/v1/notifications/devices/${uuid}/safes/${safeAddress}`;
+      mockNetworkService.get.mockImplementation((url) =>
+        url.includes(`${safeConfigUrl}/api/v1/chains/`)
```

Comment:
Same as https://github.com/5afe/safe-client-gateway-nest/pull/268#discussion_r1102693845

Reply:
Changed in [20a87a3](https://github.com/5afe/safe-client-gateway-nest/pull/268/commits/20a87a357afa6c9e9059a2e259a0a64d3468d812)